### ISSUE TITLE
validate_jwt.py does work with App Engine standard using Python 3

### ISSUE THREAD

**Issue by johnalowry** [GoogleCloudPlatform/python-docs-samples#4385]
## In which file did you encounter the issue?

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iap/README.md

### Did you change the file? If so, how?

No

## Describe the issue

The doc states: "validate_jwt is not compatible with App Engine standard environment".

I verified that validate_jwt.py does work with App Engine standard using Python 3 (which can't use App Engine Users API).

These are the steps I followed:

* Set up python3 app on App Engine standard: https://cloud.google.com/appengine/docs/standard/python3/quickstart
*  Set up IAP: https://cloud.google.com/iap/docs/app-engine-quickstart#enabling_iap
* Add the following to requirements.txt for the standard_python3/hello_world/ app:

```
google-auth==1.18.0
requests==2.24.0
cryptography==2.9.2
```

* Run: pip install -r requirements.txt
* Edit main.py

```
17,18d16
< from flask import request
< from validate_jwt import validate_iap_jwt_from_app_engine
29,30c27
< iap_jwt = request.headers.get('X-Goog-IAP-JWT-Assertion')
< return str(validate_iap_jwt_from_app_engine(iap_jwt, 1234567890, 'myproject'))
```

* gcloud app deploy
* Verified the output

**Comment by engelke** [comment]
Three years ago, when this sample was written, it could not run in the App Engine standard for Python 2.7 environment. Now it can (see https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/appengine/standard/migration/incoming/main.py for an example). This sample will be updated to reflect that current capability.

**Comment by engelke** [comment]
Fixed in PR #4393
